src/main/java/org/springframework/shell/table/AlignmentStrategy.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.shell.table;
+
+/**
+ * A strategy interface for performing text alignment.
+ *
+ * @author Eric Bottard
+ */
+public interface AlignmentStrategy {
```

Comment:
maybe `Aligner` instead vs. putting strategy in the interface name (seems not necessaary) as most subclasses naturally use the `Aligner` suffix.